SU-SWS/stanford_profile
# D8CORE-2572 Added and configured content_lock to prevent simultaneous edits

# READY FOR REVIEW

# Summary
- Added and configured [content_lock](https://www.drupal.org/project/content_lock) module

# Need Review By (Date)
- 4/7

# Urgency
- low

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-2572`
2. `blt drupal:update` or `blt drupal:install`
3. `drush pmu simplesamlphp_auth`
4. log in as a user and go to edit a node
5. in another browser or in an incognito window, log in as another user and edit the same node
6. verify you get a message and you can't save the form.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
